Sources/WrappedDefault.swift

```diff
@@ -54,5 +54,9 @@ public struct WrappedDefault<T: UserDefaultsSerializable> {
         // because `fetch` assumes that `registerDefault` has been called before
         // and uses force unwrap
         self._publisher = CurrentValueSubject<T, Never>(userDefaults.fetch(keyName))
+
+        self._observer = ObserverTrampoline(userDefaults: userDefaults, key: keyName) { [_publisher] in
+            _publisher.send(userDefaults.fetch(keyName))
```

Comment:
I think we have a retain cycle here, right?

`self -> observer -> block -> self._publisher`

However, I will forward-fix after merging this. 😊

Reply:
I think the capture list grabs `_publisher` directly, not `self._publisher`, so it should be ok. But I would appreciate a second look.